### ISSUE TITLE
Set Detection Environment Setup

### ISSUE THREAD

**Issue by AraiYuno** [AraiYuno/Myo-Armband-Keyboard#1]
- [x] Read in orientation & orientationEnuler files and make a list for timestamps which can be used to separate 10 inputs in EMG file. One interval for timestamp should be determined by a peak point that is present in Accelerometer to ignore any insignificant changes due to noise.
- [x] Use the timestamp list to make 10 valid inputs' list
- [x] Implement a multi-layer perception in Tensorflow using softmax to classify a data series nto different commands. At least a standard 3 level perceptron with 4 hidden units.

**Comment by AraiYuno** [comment]
Multi-layer part can be started on Feb 27, 2018

**Comment by AraiYuno** [comment]
Need a better way to refine the data input for the multi-layer perceptron.

**Comment by AraiYuno** [comment]
Multi-layer perceptron completed
